app/Http/Controllers/User/BlogController.php

```diff
@@ -48,11 +47,85 @@ public function store(Request $request)
             'title' => 'required',
             'content' => 'required',
         ]);
-        $blog = new Blog;
-        $blog->user_id = $this->defineUser()->id;
-        $blog->title = $request->title;
-        $blog->content = $request->content;
-        $blog->save();
-        return redirect()->back();
+        
+        try {
+            $blog = new Blog;
+            $blog->user_id = $this->defineUser()->id;
+            $blog->title = $request->title;
+            $blog->content = $request->content;
+            $blog->save();
+
+            return redirect()->back()->with('message', 'Create new blog success !');
+        } catch (Exception $e) {
+            return redirect()->back()->withErrors('Create blog error');
+        }
+    }
+
+    /**
+     * Delete one blog.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function destroy($id)
+    {
+        if ($this->defineUser()->can_delete === Blog::IS_TRUE) {
+            $blog = Blog::find($id);
+            if (isset($blog)) {
+                try {
+                    $blog->delete();
+                } catch (Exception $e) {
+                    return redirect()->back()->withErrors('Delete blog error');
+                }
+            } else {
+                return redirect()->back()->withErrors('Delete blog error');
+            }
+        }
+
+        return redirect()->back()->with('message', 'Delete blog success !');
```

Comment:
create a small if with direct return at begin of the function is better than a big if then small return

Reply:
I fixed